app/views/internal/configs/show.html.erb

```diff
@@ -11,16 +11,74 @@
   </h2>
 </div>
 
+<div class="row my-3" id="getStarted">
+  <div class="card w-100">
+    <div class="card-header" id="getStartedHeader" data-toggle="collapse"
+      data-target="#getStartedBodyContainer" aria-expanded="true" aria-controls="getStartedBodyContainer">
+      <h2 class="d-inline">Get Started</h2>
+      <button class="btn btn-secondary float-right" type="button">
+        Toggle Get Started
+      </button>
+    </div>
+
+    <div id="getStartedBodyContainer" class="collapse show hide p-3" aria-labelledby="siteConfigHeader">
```

Comment:
We can probably move this into a partial. Once I get feedback and I'm polishing I can do this.